index.html

```diff
@@ -2275,68 +2213,146 @@ <h2>
 residual risks (such as the risks from compromise in a related
 protocol, incorrect implementation, or cipher) after threat
 mitigation has been deployed.
-        </li>
+      </p>
 
-        <li>
+      <p>
 This section MUST provide integrity protection and update
 authentication for all operations required by
 Section <a href="#did-operations"></a>.
-        </li>
+      </p>
 
-        <li>
+      <p>
 Where <a>DID methods</a> make use of peer-to-peer computing resources
 (such as with all known <a>DLTs</a>), the expected burdens of those
 resources SHOULD be discussed in relation to denial of service.
-        </li>
+      </p>
 
-        <li>
+      <p>
 Method-specific endpoint authentication MUST be discussed. Where
 <a>DID methods</a> make use of <a>DLTs</a> with varying network topology, sometimes
 offered as "light node" or "<a href="https://en.bitcoin.it/wiki/Thin_Client_Security">thin client</a>"
 implementations to reduce required computing resources, the security
 assumptions of the topology available to implementations of the <a>DID
 method</a> MUST be discussed.
-        </li>
+      </p>
 
-        <li>
+      <p>
 <a>DID methods</a> MUST discuss the policy mechanism by which <a>DIDs</a> are
 proven to be uniquely assigned. A <a>DID</a> fits the functional definition
 of a URN as defined in [[RFC8141]] &mdash; a persistent identifier that is
 assigned once to a resource and never reassigned. In a security
 context this is particularly important since a <a>DID</a> may be used to
 identify a specific party subject to a specific set of authorization
 rights.
-        </li>
+      </p>
 
-        <li>
+      <p>
 <a>DID methods</a> that introduce new authentication <a>service endpoint</a>
-types (Section <a href="#service-endpoints"></a>) SHOULD consider the
+types (see <a href="#service-endpoints"></a>) SHOULD consider the
 security requirements of the supported authentication protocol.
-        </li>
-      </ol>
+      </p>
+
+    </section>
+
+    <section class='normative'>
+      <h2>
+Privacy Requirements
+      </h2>
+
+      <p>
+<a>DID method</a> specifications MUST include their own Privacy
+Considerations sections, if only to point to
+<a href="#privacy-considerations"></a>.
+      </p>
+
+      <p>
+The <a>DID method</a> specification's Privacy Considerations section
+MUST discuss any subsection of section 5 of [[RFC6973]] that could
+apply in a method-specific manner. The subsections to consider are:
+surveillance, stored data compromise, unsolicited traffic, misattribution,
+correlation, identification, secondary use, disclosure, exclusion.    
+      </p>
     </section>
 
+  </section>
+
+  <section class='normative'>
+    <h1>
+DID Resolvers
+    </h1>
+
+    <p>
+A <a>DID resolver</a> is a software or hardware component with an API for resolving
+<a>DIDs</a> of at least one <a>DID method</a>. It
+executes the corresponding <a>DID method</a>'s <a href="#read-verify">Read</a> operation to
```

Comment:
There's another PR that edits this paragraph... The other wording is preferred, when the conflict comes to resolution.

Reply:
I didn't edit this one, it just moved because of the rearranging.